activemq-karaf/src/main/resources/features-core.xml

```diff
@@ -25,7 +25,7 @@
 
     <!-- Bundles needed if only client will be deployed in the container -->
     <feature name="activemq-client" description="ActiveMQ client libraries" version="${project.version}" resolver="(obr)" start-level="50">
-        <feature version="[4,5)">spring</feature>
+        <feature version="[4,6)">spring</feature>
```

Comment:
Are you sure it works fine with spring 5 ? I gonna try anyway.

Reply:
@jbonofre I started the spring broker + web console with Karaf 4.3.0-SNAPSHOT and it seemed to work OK, I didn't investigate any further

Reply:
OK thanks. Let me double check.